### PR TITLE
Add ezTilt environment for pos-rails, ez-rails, and ezManage (FAJITAPOS-1206)

### PR DESCRIPTION
**Pull request by jasper-chen**
FAJITAPOS-1206

This adds a tilt environment to test direct entry orders that get sent to pos-rails.

• Tested that pos-complete still works
• Tested that pos-with-ezmanage works

### CODE REVIEW

File: pos-complete/Tiltfile.base [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## ez-rails
```
Comment (djcp) [inline pos-complete/Tiltfile.base:1]: Nice extraction here!

File: pos-with-ezmanage/Tiltfile [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+)
+k8s_resource('pos-consumer',
+  resource_deps=['pos', 'kafka-tilt', 'seed-pos-db']
+)
+k8s_resource('pos',
+  port_forwards='21070',
+  resource_deps=['pos-db', 'pos-redis', 'kafka-tilt']
+)
+
+seed_pos()
```
Comment (imacomber) [inline pos-with-ezmanage/Tiltfile:27]: Could all the additive calls be included in the `pos-complete/Tiltfile.base` to reduce duplication?

**Comment by djcp** [comment]
It'd be create if you could squash your commits before merging, @jasper-chen

File: pos-with-ezmanage/Tiltfile (hunk 2 above)
Comment (djcp) [reply pos-with-ezmanage/Tiltfile:27]: We've found that different clusters have different dependency requirements, so this is one place where maybe it's not actually duplication.

**Comment by jasper-chen** [comment]
@mikowitz thank you for pairing on this with me! I learned a lot throughout the whole process.

File: pos-with-ezmanage/Tiltfile (hunk 2 above)
Comment (djcp) [reply pos-with-ezmanage/Tiltfile:27]: https://www.sandimetz.com/blog/2016/1/20/the-wrong-abstraction

I feel like reasoning about a cluster config's dependency graph by poring through includes would be the wrong kind of abstraction here. This is how I'm looking at it for now, at least.

Comment (djcp) [reply pos-with-ezmanage/Tiltfile:27]: The other stuff is mostly build related and unchanging from cluster to cluster.

**Pull request merged**